src/test/java/testsuite/failover/FailoverIntegrationTest.java

```diff
@@ -193,18 +177,24 @@ public void test1_2_failFromWriterToNewWriter_failOnConnectionBoundObjectInvocat
   @Test
   public void test1_3_writerConnectionFailsDueToNoReader()
       throws SQLException, InterruptedException {
-    final String initalWriterId = INSTANCE_ID_1;
+    final String initalWriterId = instanceIDs[0];
 
-    testConnection = connectToWriterInstance(initalWriterId);
+    Properties props = new Properties();
+    props.setProperty(PropertyKey.USER.getKeyName(), TEST_USERNAME);
+    props.setProperty(PropertyKey.PASSWORD.getKeyName(), TEST_PASSWORD);
+    props.setProperty(PropertyKey.connectTimeout.getKeyName(), CONNECT_TIMEOUT_VAL);
+    props.setProperty(PropertyKey.socketTimeout.getKeyName(), SOCKET_TIMEOUT_VAL);
+    props.setProperty(PropertyKey.socketFactory.getKeyName(), testsuite.UnreliableSocketFactory.class.getName());
+    props.setProperty(PropertyKey.failoverTimeoutMs.getKeyName(), "10000");
+    testConnection = connectToWriterInstance(initalWriterId, props);
 
     // Crash all reader instances (2 - 5).
-    startCrashingInstanceAndWaitUntilDown(INSTANCE_ID_2);
-    startCrashingInstanceAndWaitUntilDown(INSTANCE_ID_3);
-    startCrashingInstanceAndWaitUntilDown(INSTANCE_ID_4);
-    startCrashingInstanceAndWaitUntilDown(INSTANCE_ID_5);
+    for (int i = 2; i < instanceIDs.length; i++) {
```

Comment:
should this start from i = 1? The test originally crashes all but the first instance I think

Reply:
Good catch. Fixed.